tests/console/validate_ext4_fs.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate ext4 filesystem and system partitioning
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+
+sub run {
+
+    select_console "root-console";
+
+#Validate that swap is created and /home is on seperate partition. 
+script_run("df -PH /");
+script_run("df -PH /home");
+save_screenshot;
```

Comment:
I think you will have the output in one result box in openQA, I don't think you need the screenshot here, right?